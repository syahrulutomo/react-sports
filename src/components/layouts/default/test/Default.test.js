import React from 'react';
import {cleanup} from '@testing-library/react';
import { LayoutDefault } from '../index'
import ShallowRenderer from 'react-test-renderer/shallow';

afterEach(cleanup)

const children = (
  <div></div>
)

it('renders correctly Layout Default component', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<LayoutDefault children={children}/>);
  const result = renderer.getRenderOutput();

  expect(result).toMatchSnapshot();
});