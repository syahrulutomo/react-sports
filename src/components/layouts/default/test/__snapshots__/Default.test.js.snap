// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Layout Default component 1`] = `
<main
  className="app-layout--default"
>
  <div />
</main>
`;
